From 0dc5f4bcc7c95bffed4855cba537ccf8a1cad876 Mon Sep 17 00:00:00 2001
From: Victor Seva <linuxmaniac@torreviejawireless.org>
Date: Fri, 21 Mar 2014 21:52:03 +0100
Subject: [PATCH] parser: add internal flag FL_BODY_MULTIPART

This will be set if the body has been converted to multipart/mixed
or a new part has been appended to it. This will trigger the core
to check and fix the boundaries of the body.
---
 parser/msg_parser.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/parser/msg_parser.h b/parser/msg_parser.h
index 724ebef..f25bec9 100644
--- a/parser/msg_parser.h
+++ b/parser/msg_parser.h
@@ -121,6 +121,7 @@ typedef enum request_method {
 #define FL_USE_UAC_TO        (1<<14)  /* take TO hdr from UAC instead of UAS */
 #define FL_TM_RPL_MATCHED    (1<<15)  /* tm matched reply already */
 #define FL_RPL_SUSPENDED     (1<<16)  /* for async reply processing */
+#define FL_BODY_MULTIPART    (1<<17)  /* body modified is multipart */
 
 /* WARNING: Value (1 << 28) is temporarily reserved for use in kamailio call_control
  * module (flag  FL_USE_CALL_CONTROL )! */
-- 
2.0.0.rc0

